flake.nix

```diff
@@ -281,7 +330,23 @@
         # the old build system is gone and we are back to one build
         # system, we should reenable this.
         #perlBindings = self.hydraJobs.perlBindings.${system};
-      } // devFlake.checks.${system} or {}
+      }
+      # Add "passthru" tests
+      // flatMapAttrs {
+          "" = nixpkgsFor.${system}.native;
+          # FIXME: Static build on darwin currently broken
+          # "static-" = nixpkgsFor.${system}.static;
```

Comment:
I thought it might have been https://github.com/NixOS/nixpkgs/pull/316074, but apparently not, because that's a backport and I get the same error on `nixos-unstable`, where that change originates.
The error message is:

```
       > FileNotFoundError: [Errno 2] No such file or directory: 'lipo'
       >
       > ERROR: Unhandled python OSError. This is probably not a Meson bug, but an issue with your build environment.
```

Reply:
Found it:
- https://github.com/NixOS/nixpkgs/issues/320448

Will update the comment.